build.cmd

```diff
@@ -26,26 +19,9 @@ setlocal enabledelayedexpansion
   call "%~dp0.\dotnet-install.cmd" || exit /b 1
 
   set procedures=
-
-  if not defined lv_api_only (
-    set procedures=!procedures! build_procdomain
```

Comment:
@brianrob @dpodder Could you please take a look? I have updated all sources based on feedback, added some error checking, bug fixes, updated readme, remove runner projects and disabled the Linux build script (there is not CLI to be built, and I would need to update the groovy file first, in future PR)